docs/development.md

```diff
@@ -6,15 +6,79 @@
 
 ### Cloning the repo
 
-TODO: Describe how to clone, folder structure etc.
+To create a local copy of Presidio repository, follow [Github instructions](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository)
+on how to clone a project using git.
+The project is structured so that:
+ - Each Presidio service has a designated directory. In it, the source code for its
+logic and tests, as well as the source code for serving it as an HTTP service and
+as a python package.
+ - In the project root directory, you will find common code for using, serving and testing Presidio
+as a cluster of services, as well as CI/CD pipelines codebase and documentation.
 
 ### Setting up Pipenv
 
-TODO: Add Pipenv documentation from V1.
+[Pipenv](https://pipenv.pypa.io/en/latest/) is a Python workflow manager, handling
+dependencies and environment for Python packages. It is used by each Presidio service
+as the dependencies manager, to avoid dependencies issues when switching between different
```

Comment:
I think this sentence can be shorter something like: 
A dependency manager to align the versions.

Reply:
Shortened